src/ape/managers/chain.py

```diff
@@ -121,9 +121,15 @@ def query(
             pd.DataFrame
         """
 
+        if start_block < 0:
```

Comment:
maybe add a test or two?

might be unrelated, but also wondering about negative step

Reply:
Handling negative steps would be a bit of work, I'll skip that for now.

Adding some tests

Reply:
see [1bc374d](https://github.com/ApeWorX/ape/pull/829/commits/1bc374dc1d4afac799123948ea1f8c26008eec7a)